aws/v12.0.0/release.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v12.0.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.2.3
+  - componentVersion: 0.9.0
+    name: cert-manager
+    version: 1.0.8
+  - name: chart-operator
+    version: 0.12.4
+  - componentVersion: 1.16.5
+    name: cluster-autoscaler
+    version: 1.16.0
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 0.7.2
+    name: external-dns
+    version: 1.2.2
+  - componentVersion: 3.5.0
+    name: kiam
+    version: 1.3.0
+  - componentVersion: 1.9.5
+    name: kube-state-metrics
+    version: 1.1.0
+  - componentVersion: 0.3.3
+    name: metrics-server
+    version: 1.1.0
+  - name: net-exporter
+    version: 1.9.0
+  - componentVersion: 0.18.1
+    name: node-exporter
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-cni
+    version: 1.6.0
+  - name: aws-operator
+    version: 8.7.3
+  - name: calico
+    version: 3.12.3
```

Comment:
Can we bump this to 3.13+ for https://github.com/projectcalico/kube-controllers/pull/459?

Reply:
I will be bumping many version here, this is still WIP :)

Reply:
what about 3.14 and 3.15 ? should we try that as well?

Reply:
Cool, thanks. Just didn't want it to get lost. I'll add it to the releases board as well. We're using 3.14 in KVM 12.0 so I think that's safe. There could be some changes required for 3.15 or maybe it will "just work".